framework/src/play/Play.java

```diff
@@ -373,62 +346,16 @@ public static <T extends PlayPlugin> T plugin(Class<T> clazz) {
     return pluginCollection.getPluginInstance(clazz);
   }
 
-  private void loadModules(VirtualFile appRoot) {
-    File localModules = Play.getFile("modules");
-    if (localModules.exists() && localModules.isDirectory()) {
-      for (File module : localModules.listFiles()) {
-        if (module == null || !module.exists()) {
-          logger.error("Module {} will not be loaded because {} does not exist", module.getName(), module.getAbsolutePath());
-        }
-        else if (module.isDirectory()) {
-          addModule(appRoot, module.getName(), module);
-        }
-        else {
-          File modulePath = new File(IO.readContentAsString(module, UTF_8).trim());
-          if (!modulePath.exists() || !modulePath.isDirectory()) {
-            logger.error("Module {} will not be loaded because {} does not exist", module.getName(), modulePath.getAbsolutePath());
-          }
-          else {
-            addModule(appRoot, module.getName(), modulePath);
-          }
-        }
-      }
-    }
-  }
-
-  /**
-   * Add a play application (as plugin)
-   *
-   * @param appRoot the application path virtual file
-   * @param name    the module name
-   * @param path    The application path
-   */
-  private void addModule(VirtualFile appRoot, String name, File path) {
-    VirtualFile root = VirtualFile.open(path);
-    modules.put(name, root);
-    if (root.child("app/views").exists()
-        || (usePrecompiled && appRoot.child("precompiled/templates/from_module_" + name + "/app/views").exists())) {
-      templatesPath.add(root.child("app/views"));
-    }
-    if (root.child("conf/routes").exists()
-        || (usePrecompiled && appRoot.child("precompiled/templates/from_module_" + name + "/conf/routes").exists())) {
-      modulesRoutes.put(name, root.child("conf/routes"));
-    }
-    roots.add(root);
-    if (!name.startsWith("_")) {
-      logger.info("Module {} is available ({})", name, path.getAbsolutePath());
-    }
-  }
-
   /**
-   * Search a VirtualFile in all loaded applications and plugins
+   * Search a File in the application
    *
    * @param path Relative path from the applications root
    * @return The virtualFile or null
    */
   @Nullable
-  public static VirtualFile getVirtualFile(String path) {
-    return VirtualFile.search(roots, path);
+  public static File getVirtualFile(String path) {
```

Comment:
Maybe rename this then as well.

I know "get" is very common in Java, even when it is not a getter; but we see in most places these meaningless "get" prefixes get dropped.

Reply:
There's quite a bit of filesystem related methods in this class, I'd put them when possible in an `fs` package (like the `db` and the `vfs` package).

Reply:
Thanks, I renamed `Play.getVirtualFile()` to just `Play.file()`